### PR TITLE
weston: update to 13.0.0.

### PR DESCRIPTION
**Pull request by Calandracas606**
#### Testing the changes
- I tested the changes in this PR: **YES**
- Tested the colour management, and HDR features using DRM back-end
- Only tested the DRM, and nested wayland back-ends

#### Local build testing
- I built this PR locally for my native architecture, x86_64-glibc
- I built this PR locally for these architectures (if supported. mark crossbuilds):
  - x86_64-musl
  - i686
  - aarch64-musl
  - armv6l-musl (crossbuild)
  - armv7l (crossbuild)
  - aarch64 (crossbuild)

### CODE REVIEW

File: srcpkgs/removed-packages/template [hunk 1]
```diff
@@ -695,6 +695,7 @@ replaces="
  vte290<=0.36.5_5
  vte<=0.28.2_16
  w_scan<=20170107_1
+ weston-colord<=9.0.0
```
Comment (classabbyamp) [inline srcpkgs/removed-packages/template:698]:
```suggestion
 weston-colord<=9.0.0_1
```

File: srcpkgs/removed-packages/template [hunk 2]
```diff
@@ -695,6 +695,7 @@ replaces="
  vte290<=0.36.5_5
  vte<=0.28.2_16
  w_scan<=20170107_1
+ weston-colord<=9.0.0_1
```
Comment (sgn) [inline srcpkgs/removed-packages/template:698]: This should go to `weston` instead.

File: srcpkgs/weston/template [hunk 3]
```diff
@@ -1,38 +1,34 @@
...
+ $(vopt_if vaapi 'glu-devel libva-devel')"
 short_desc="Reference implementation of a Wayland compositor"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="MIT"
 homepage="https://wayland.freedesktop.org/"
 distfiles="https://wayland.freedesktop.org/releases/${pkgname}-${version}.tar.xz"
-checksum=5cf5d6ce192e0eb15c1fc861a436bf21b5bb3b91dbdabbdebe83e1f83aa098fe
-system_groups="weston-launch"
+checksum=52ff1d4aa2394a2e416c85a338b627ce97fa71d43eb762fd4aaf145d36fc795a
 lib32disabled=yes
```
Comment (sgn) [inline srcpkgs/weston/template:21]:
```suggestion
lib32disabled=yes
replaces="weston-colord>=0"
```

File: srcpkgs/removed-packages/template (hunk 2 above)
Comment (Calandracas606) [reply srcpkgs/removed-packages/template]: should I squash the removed weston colord into the main update weston commit?

**Comment by Calandracas606** [comment]
What can I do to fix the CI failures? I believe its a false positive, since weston-colord is replaced

**Comment by Calandracas606** [comment]
@skmpz FYI this PR has changes to your lcms2 package

**Comment by sgn** [comment]
> What can I do to fix the CI failures? I believe its a false positive, since weston-colord is replaced

It's not false positive. Your change will stage the repository.

**Pull request merged**